### ISSUE TITLE
Wow JS is causing text to strobe at the top of website

### ISSUE THREAD

**Issue by jeremyrschroedercd2** [matthieua/WOW#284]
My content that is usually animated in the middle of the page is showing at the top and stobes ( flash on and off at the very top of the page). I don't get it all the time - it is hard to replicate.

**Comment by dnk588** [comment]
Same issue! Happening in Chrome only. And only for special elements, for example:

<div class="quote wow fadeInUp">
Speed is important
<div class="autor">
<div class="yellow-line"></div>
<span class="name">Autor name</span>
</div>
</div>

So Autor name will apear on top. Also it's happening in wordpress, so wordpress wrapping span in p, maybe this is also the reason. Will look for more details.

**Comment by jeremyrschroedercd2** [comment]
Here is the page i was having trouble on - if that helps:  http://portal1.cd2learning.com/home.aspx
It pretty easy to find the problem though you might have to scroll all the way down before you notice it - no luck with understanding a solution though:

**Comment by dnk588** [comment]
In your example i see text in blocks disappearing for a second, going on top, then appearing again at the place it mentioned to be.
It don't looks solved)
Using chrome Version 58.0.3029.110 (64-bit) on mac

**Comment by jeremyrschroedercd2** [comment]
Yes not solved. I removed the Wow animation on the live site - this is just a testing site as I am still not finding the cause. I thought maybe showing you additional example of the problem might help. 

I am on also Using chrome Version 58.0.3029.110 (64-bit) on mac

**Comment by jeremyrschroedercd2** [comment]
It is just a Chrome V58 issue, I just confirmed is also does this on a PC and doesn't do it on any other old versions of Chrome - fyi, guessing chrome may have dropped support a CSS3 parameter wow might be using.

**Comment by dnk588** [comment]
Yes, you are right, on previous versions of chrome everything worked perfectly, just recently this problem started to appear.

**Comment by pantaluna** [comment]
I am using "animate.css": "~3.5.2", with "wowjs": "~1.1.2" and having exactly the same issue but only when using the annotation "wow fadeInUp" and somehow it does not happen every time you load the page (timing issue?). The sections with annotations "wow slideInLeft" and "wow slideInRight" animate correctly.
I guess this is an issue in animate.css and not wowjs. No idea how to "debug" css animations.

**Comment by dbledayr** [comment]
I had this same issue for when a parent is animated and the contents are absolutely positioned (e.g. when you are "perfectly" centering content in a block or similar. So far it appears adding an appropriate z-index to the content that is having the issue resolves said problem.

**Comment by jeremyrschroedercd2** [comment]
So I noticed that Chrome just updated automatically for me and I no longer see the issue. The issue has _at times_ be really hard to replicate so not 100% the update fixed it but thinking it did. Can any one else confirm their issue has been resolved with the latest update?

**Comment by pantaluna** [comment]
@jeremyrschroedercd2 yes it works fine now in Chrome Version 59.0.3071.115 (Official Build) (64-bit) for Windows 10
